README.md

```diff
@@ -1,60 +1,52 @@
-# curriculum-databases-projects-template
+<a name="readme-top"></a>
 
-## Getting Started
 
-This repository includes files with plain SQL that can be used to recreate a database:
+<div align="center">
+ 
+  <img src="https://d1q6f0aelx0por.cloudfront.net/product-logos/library-postgres-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
 
-<a name="readme-top"></a>
+
+</div>
+
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be very nice if you make all your commit messages capitalized to make your pull request professional :+1: